lib/variable/string.cpp

```diff
@@ -71,6 +71,14 @@ auto apply(const DType dtype, Args &&...args) {
 }
 } // namespace
 
+std::string format_variable_compact(const Variable &variable) {
+  const auto s = to_string(variable.dtype());
+  if (variable.unit() == units::none)
+    return s + "[]";
```

Comment:
```suggestion
  if (variable.unit() == units::none)
    return s;
```

Reply:
Is this the agreed rule, that we should print nothing?
When do we print nothing, and when do we print `<no unit>`?

Reply:
We should probably print nothing everywhere.